resources/xlf/en/sql.xlf

```diff
@@ -5244,6 +5244,17 @@ Error: {1}</source>
     <trans-unit id="kerberosKinit">
       <source xml:lang="en">If you have previously connected you may need to re-run kinit.</source>
     </trans-unit>
+    <trans-unit id="runKinit">
```

Comment:
I don't think you need to update these files manually.  They should get auto generated from the source during next localization update.

Reply:
I'm used to updating the English ones in .NET, that's why did it here. 
Do we not need that usually in 'en' files in Typescript?

Reply:
My understanding is that if there are no localization resources than the strings embedded directly in the code will be used.  The localization process scans the source code and extracts the English strings, which are picked up by the loc team for translation.  So you shouldn't need to directly update the XLF files.

Reply:
That is correct. The fallback will always use the strings in the code - the xlf files are an interim step to generating the langpacks which have the localized strings used at runtime. But that is currently something that @smartguest takes care of on a regular basis so while updating these yourself isn't bad it's not something you need to do.

Reply:
I'll let it stay for now and will consider skipping those in future.